web/src/Main.hs

```diff
@@ -26,10 +26,12 @@ import Text.Pretty.Simple.Internal (Annotation (..), layoutStringAbstract)
 
 #ifndef __GHCJS__
 runApp :: JSM () -> IO ()
-runApp f =
+runApp f = do
+    putStrLn "Web server running on 0.0.0.0:8000..."
```

Comment:
Out of interest, why `0.0.0.0` instead of `localhost`?

Reply:
It appears that by default warp actually does listen on all interfaces (`0.0.0.0`), and not just localhost.  So I felt this message was "more correct", although I guess some developers might not know that `0.0.0.0` means they can connect by accessing http://127.0.0.1:8000.